app/mailers/user_mailer.rb

```diff
@@ -417,12 +417,14 @@ def feedback(feedback_id)
 
   def abuse_report(abuse_report_id)
     abuse_report = AbuseReport.find(abuse_report_id)
+    @username = abuse_report.username
     @email = abuse_report.email
     @url = abuse_report.url
+    @summary = request.summary
```

Comment:
I think this needs to be
```suggestion
    @summary = abuse_report.summary
```
which should fix the current error in `spec/mailers/user_mailer_spec.rb` (although the text in those tests probably needs updating before they pass).